### PR TITLE
77 Update encrypted npm key

### PR DESCRIPTION
**Pull request by allishultes**
**Is your Pull Request request related to another [issue](https://github.com/bbc/digital-paper-edit-storybook/issues) in this repository ?**      
#77 

**Describe what the PR does**    

- [x] Replaces Eimi's npm token with mine (see https://github.com/bbc/digital-paper-edit-storybook/pull/80#issuecomment-824734113)
- [x] Adds CHANGELOG.md

**State whether the PR is ready for review or whether it needs extra work**    

Ready for review

This has been validated by merging into the release branch (see comment linked above). 

**Additional context**    

To be updated with newslabs-jenkins creds when the bug regarding access is sorted out.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 before_deploy:
-  - npm run build
-  - npm run cp
-  - cd dist
+- npm run build
+- npm run cp
+- cd dist
 deploy:
   provider: npm
   email: eimi.okuno@bbc.co.uk
```
Comment (emettely) [inline .travis.yml:11]: This email is still mine, does it work without it??

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+    secure: pD8nVakQjdQgXYIXHaatQeHOsodGXVuLOPmmZjHlFLvS0fsMeB1YfO4N730mwrrjqgjO9taBxzhHo79mB8VwUntdJML3X8gqCZVvr76XEUG/ivHAbpxBLujLO0bakboeGa4BROJIh0bZWreLRzOe+KVuPo2L1ZYw5/8u1rs4bJSG6rqsNiQMhbEBcHMy/Ge9NH5OkYvKlHvU34bYugCvaMgyCgUCmq3OdfAgNYoepAg5tm2wdUPSOgMNRad3Nh/AAd/8qn/ho4JeL8ca1JceEy8FvPJVA+8FeFlkOe9xnrulLBZw2My5+HaeIhSDXgZ8ghudMBHcjHMtHohP0ZDJUi6pesih0/O57XG1uBlXK0EswFut2K9pE4/D8+hKi0XR1ZwM4bg/vT9IMyFxuf3MxKdlm9kfZTtBPyhdC6cInIFn/eI4fzKCD3pyCoTD2WaZHqs6nYSPfK36v1M829WaK4WUl903fvPiPqDEAerJIarLkVzvU8B9d9FqQTxOJvdqvgNuzaEKKHtmIo40EZMaJYpJdTwwUKLriOYtstkYVsd2H/r7Ovh/ZN6lFRXbsW1PyEEERjT/X10LiJ7DnzomPPxeJRc7iMEqb8ZMFU5NTSd+YJCq2itans0udxlstoIFKeU68wp2e1AXINj70zBbg3F0wgeAZ0hqZR5+rhXSGWA=
   on:
     branch: release
 after_deploy:
-  - cd ..
+- cd ..
```
Comment (emettely) [inline .travis.yml:18]: Probably needs to re-add the spaces, as it's clearer

File: .travis.yml [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
-  - "12.18.4"
+- 12.18.4
 before_install: npm i -g rimraf
 before_deploy:
-  - npm run build
-  - npm run cp
-  - cd dist
+- npm run build
+- npm run cp
+- cd dist
```
Comment (emettely) [inline .travis.yml:8]: Re-add spacing

**Comment by emettely** [comment]
Probably worth pulling in the changes to release to test it :) as you mentioned

File: .travis.yml (hunk 1 above)
Comment (allishultes) [reply .travis.yml:11]: shoot, good shout

**Review by emettely — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [ 1.22.1 ] - 2021-04-20
+### Changed
+- Updated travis.yml to replace Eimi's npm credentials with our Jenkins credentials
```
Comment (tamsingreen) [inline CHANGELOG.md:10]: Would be good to get a ref to the issue number on this line eg "I77: Updated Travis...."

**Review by tamsingreen — APPROVED** [review]
Looks good Alli, great work. The only unknown here is whether this NPM user has write permissions 😬 but we may be able to fix that if not

**Comment by allishultes** [comment]
> Looks good Alli, great work. The only unknown here is whether this NPM user has write permissions 😬 but we may be able to fix that if not

There are some issues with the permissions, it looks like - Nik added the account to the BBC News Labs team but still getting permissions issues. 

Screenshot 2021-04-20 at 12 43 29

**Comment by niklasR** [comment]
Hi,
I've just ran `npm access grant read-write @bbc:news-labs @bbc/digital-paper-edit-storybook` and it looks like it should have write access now:
```
➜  ~ npm access ls-collaborators @bbc/digital-paper-edit-storybook | grep newslabs-jenkins
  "newslabs-jenkins": "read-write",
```

@allishultes can you try again?

**Comment by allishultes** [comment]
@tamsingreen @niklasR and I have tried to grant the Jenkins account read-write access for the storybook package but this isn't working. Mysteriously I'm able to publish with my npm credentials, so for now these are set in Travis. To be updated when npm responds to Nik. 

The Travis deployment now works, as validated by https://github.com/bbc/digital-paper-edit-storybook/pull/85.

**Pull request merged**